### PR TITLE
Add Milesight hotspot

### PR DESCRIPTION
**Pull request by Milesight-IoT**

### CODE REVIEW

File: src/makers/Milesight/index.ts [hunk 1]
```diff
@@ -0,0 +1,118 @@
+import bobcat from './bobcat'
```
Comment (matthewcarlreetz) [inline src/makers/Milesight/index.ts:1]: This file belongs in `src/makers/index.ts`

`src/makers/Milesight/index.ts` Should look something like...
```ts
import antennas from './antennas'
import hotspots from './hotspots'

export default {
  antennas,
  hotspots,
  id: 9999, // your maker id
  supportEmail: 'support@milesight.asdf.com', // your support email
}
```

**Comment by tyler-whitman** [comment]
@Milesight-IoT you forgot to complete step 4 of https://github.com/helium/hotspot-app/tree/main/src/makers#adding-a-new-maker

**Comment by Milesight-IoT** [comment]
Thanks Tyler.

**Comment by cokes518** [comment]
do you want to close this PR then @Milesight-IoT ?

**Pull request closed without merging**